### ISSUE TITLE
Dependency Error: jekyll-remote-theme is not installed

### ISSUE THREAD

**Issue by 20avva** [jekyll/jekyll#6660]
- [x] I believe this to be a bug, not a question about using Jekyll.
- [x] I updated to the latest Jekyll (or) if on GitHub Pages to the latest `github-pages`
- [x] I ran `bundle exec jekyll doctor` to check my configuration. **didn't work, gave same error as `bundle exec jekyll build`**
- [x] I read the CONTRIBUTION file at https://jekyllrb.com/docs/contributing/

---
- [x] I am on (or have tested on) ***Debian/Ubuntu*** GNU/Linux

---

- [x] I was trying to build.
- [x] It was another bug.

## My Reproduction Steps

Ran `bundler exec jekyll build --watch`

Output:
```
Configuration file: /projects/20avva.github.io/_config.yml
  Dependency Error: Yikes! It looks like you don't have jekyll-remote-theme or one of its dependencies installed. In order to use Jekyll as currently configured, you'll need to install this gem. The full error message from Ruby is: 'Could not open library 'libcurl': libcurl:
cannot open shared object file: No such file or directory. Could not open library 'libcurl.so': libcurl.so: cannot open shared object file: No such file or directory. Could not open library 'libcurl.so.4': libcurl.so.4: cannot open shared object file: No such file or director
y' If you run into trouble, you can find helpful resources at https://jekyllrb.com/help/!
jekyll 3.6.2 | Error:  jekyll-remote-theme
```

I added `gem "jekyll-remote-theme"` to `Gemfile`, ran `bundle update` and after that failing I did some spelunking online about this before filing this issue, and then changed `theme: jekyll-theme-bootstrap20avva` (its my own custom theme) to `remote_theme: jekyll-theme-bootstrap20avva`

## The Output I Wanted

I wanted it to work. 😢

**Comment by 20avva** [comment]
When I uncomment `gem "jekyll"` and instead have `gem "github-pages", group: :jekyll_plugins` it breaks.
In other words, I think it's my github-pages gem which is causing the problem

**Comment by pathawks** [comment]
Duplicate of https://github.com/benbalter/jekyll-remote-theme/issues/17 ?

**Comment by 20avva** [comment]
Not really, this is happening on Ubuntu 16.04.3 LTS (specifically a codenvy stack) but I’ll give it a shot when I get back to my computer.

**Comment by pathawks** [comment]
This is probably an issue for [jekyll-remote-theme](https://github.com/benbalter/jekyll-remote-theme); I'm not sure there's anything we can do from here to fix this.

Closing for now, but please do post back here if you figure things out.

**Comment by cetinajero** [comment]
Hi @pathawks @20avva

Actually, for what I know, it is a dependency from `Typhoeus` (i.e. a libcurl wrapper). `jekyll-remote-theme` relies on `Typhoeus ` to pull up the remote theme.

The problem that you are having @20avva its because on `github-pages` gem v171 it was added `jekyll-remote-theme` by default so (you can see it here: [github-pages v171](https://github.com/github/pages-gem/releases/tag/v171))... if you run v171 and up you will have to install `libcurl` even if you aren't using `jekyll-remote-theme`.

I had reported this issue on [github/github-pages](https://github.com/github/pages-gem/) repository, you can see the issue here: https://github.com/github/pages-gem/issues/509 and there is actually a PR on progress to fix this that you can see here: https://github.com/github/pages-gem/pull/510

Cheers!

**Comment by 20avva** [comment]
Thanks!

**Comment by ketozhang** [comment]
@cetinajero 

Worked for me thank you! Hopefully someone will ping here when the PR gets pushed.

**Comment by granbom** [comment]
Dunno if this will help anyone.
I got this same error on a freshly reinstalled WSL (Ubuntu 16.04.3). 
Solved it with:
```
sudo apt install libcurl4-openssl-dev
```

**Comment by 20avva** [comment]
I think I got it fixed by upgrading to Ruby 2.5.0 through rbenv/rbenv

**Comment by cetinajero** [comment]
PR https://github.com/github/pages-gem/pull/510 merged 👍
